app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -241,12 +243,18 @@ private void displayMediaDetails() {
         compositeDisposable.add(disposable);
     }
 
+    // The imageSpacer is Basically a transparent overlay for the SimpleDraweeView 
+    // which holds the image to be displayed( moreover this image is out of
+    // the scroll view )
```

Comment:
Can we write javadocs in /**/ format. You can check other methods in our code for having an example.